aspnetcore/security/data-protection/includes/introduction7.md

```diff
@@ -1,5 +1,8 @@
 :::moniker range="< aspnetcore-8.0"
 
+> [!WARNING]
+> Data configuration keys in this version of .NET do ***NOT*** maintain data consistency by default in web farms and [Azure Container Apps](/azure/container-apps/overview). See the the [.NET 8 version or later of this document](/aspnet/core/security/data-protection/introduction?view=aspnetcore-800.0&preserve-view=true) for information on how to maintain data consistency in web farms and Azure Container Apps.
```

Comment:
I think this page is more relevant if people are trying to make things work on ACA: https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps?view=aspnetcore-7.0&tabs=login-azure-cli